migrations/20240112082648_init.up.sql

```diff
@@ -0,0 +1,31 @@
+CREATE TABLE IF NOT EXISTS users (
+                                     id SERIAL PRIMARY KEY,
+                                     login VARCHAR(255) NOT NULL UNIQUE,
+                                     password VARCHAR(255) NOT NULL DEFAULT '',
+                                     balance INT NOT NULL DEFAULT 0
+);
+
+CREATE INDEX idx_user_login ON users (login);
+
+CREATE TABLE IF NOT EXISTS orders (
+                                      code VARCHAR(255) NOT NULL PRIMARY KEY,
+                                      uploaded_at timestamp with time zone default current_timestamp,
+                                      status VARCHAR(255) NOT NULL DEFAULT '',
+                                      user_id INT REFERENCES users(id)
+);
+
+CREATE INDEX idx_order_status ON orders (status);
+CREATE INDEX idx_order_user_id ON orders (user_id);
+
+CREATE TABLE bonus_transactions (
+                                    order_code VARCHAR(255),
```

Comment:
```suggestion
                                    order_code VARCHAR(255) NOT NULL,
```
какой смысл делать запись бонуса который не относится ни к одному заказу?